src/index.ts

```diff
@@ -465,16 +473,16 @@ export async function emitTelemetryOnErrorOutsideHandler(
       },
       startTime,
     };
-    const tracerWrapper = new TracerWrapper();
+    const config = getConfig();
+    const tracerWrapper = new TracerWrapper(config);
```

Comment:
I don't understand why is this change necessary? This method is only called when there is an error attempting to initialize the handler.